ServerWhitelist.yml

```diff
@@ -2202,6 +2202,10 @@ AllowedPrefixes:
     - https://mega.nz/file/PyIlHT5T#oQlM4WE0JGkSOyCz2UvpQQsg7dHgsUbvgiFTX7ZxIlc #Buxom Wench Yuriana 1.56 Beta Kozuke Hajime open public version. Patreon Reference page https://www.patreon.com/posts/happy-holidays-95158589
     - https://mega.nz/file/x7BE3aLY#UfBp2PdATkHcNET4bIDUHwwyVA4r8zTS8_misjkWA1o #Succubus Twins SE Mirror Reference Serana Dead Sexy - Official Nexus collection https://next.nexusmods.com/skyrimspecialedition/collections/gv7h2i
     - https://www.patreon.com/file?h=95158589&m=299795319 #Buxom Wench Yuriana 1.56 Beta Patch Kozuke Hajime Open Public version Patreon Reference page https://www.patreon.com/posts/happy-holidays-95158589
+    - 1kphyNUBtv0lqRn6naI_LNHuH5nwTIlTT #Tulius SMP Hair 3 806 Big Head
+    - 1nSqmX7VVAs6r7CyCDuPX6wkWvzZNROYM #Tulius SMP Hair 3 Fixed ESP
```

Comment:
Great, now please move the IDs to the section of Google Drive IDs, right beneath the other Google Drive IDs.
https://github.com/Tvirus5087/opt-out-lists/blob/patch-3/ServerWhitelist.yml#L620-L622